index-advisor.md

```diff
@@ -0,0 +1,222 @@
+
+## Remove unused indexes
+
+For v8.0.0 or later versions, you can identify inactive indexes in your workload using `sys.schema_unused_indexes` and `information_schema.tidb_index_usage`. Removing these indexes can save storage space and reduce overhead. For production systems, it is highly recommended to make the target indexes invisible first and observe the impact for one complete business cycle before permanently removing them.
+
+### Use `sys.schema_unused_indexes`
+
+The [`sys.schema_unused_indexes`](/sys-schema/sys-schema-unused-indexes.md) view identifies indexes that have not been used since the last startup of all TiDB instances. This view, based on system tables containing schema, table, and column information, provides the full specification for each index, including schema, table, and index names. You can query this view to decide which indexes to make invisible or delete.
+
+> **Warning:**
+>
+> Because the `sys.schema_unused_indexes` view shows unused indexes since the last startup of all TiDB instances, ensure that the TiDB instances have been running long enough. Otherwise, the view might show false candidates if certain workloads have not yet run. Use the following SQL query to identify the uptime of all TiDB instances.
+>
+> ```sql
+> SELECT START_TIME,UPTIME FROM INFORMATION_SCHEMA.CLUSTER_INFO WHERE TYPE='tidb';
+> ```
+
+### Use `information_schema.tidb_index_usage`
+
+The [`information_schema.tidb_index_usage`](/information-schema/information-schema-tidb-index-usage.md) table provides metrics such as selectivity buckets, last access time, and rows accessed. The following examples show queries to identify unused or inefficient indexes based on this table:
+
+```sql
+-- Find indexes that have not been accessed in the last 30 days.
+SELECT table_schema, table_name, index_name, last_access_time
+FROM information_schema.cluster_tidb_index_usage
+WHERE last_access_time IS NULL
+  OR last_access_time < NOW() - INTERVAL 30 DAY;
+
+-- Find indexes that are consistently scanned with over 50% of total records.
+SELECT table_schema, table_name, index_name,
+       query_total, rows_access_total,
+       percentage_access_0 as full_table_scans
+FROM information_schema.cluster_tidb_index_usage
+WHERE last_access_time IS NOT NULL AND percentage_access_0 + percentage_access_0_1 + percentage_access_1_10 + percentage_access_10_20 + percentage_access_20_50 = 0;
+```
+
+> **Note:**
+>
+> The data in `tidb_index_usage` might be delayed by up to five minutes, and the usage data is reset whenever a TiDB node restarts. Additionally, index usage is only recorded if the table has valid statistics.
+
+## Hypothetical indexes
+
+Hypothetical indexes (Hypo Indexes) are created using SQL comments, similar to [query hints](/optimizer-hints.md), rather than through the `CREATE INDEX` command. This approach enables lightweight experimentation with indexes without the overhead of physically materializing them.
```

Comment:
```suggestion
Hypothetical indexes (Hypo Indexes) are created using SQL comments, similar to [query hints](/optimizer-hints.md), rather than through the `CREATE INDEX` statement. This approach enables lightweight experimentation with indexes without the overhead of physically materializing them.
```